api/integrations/rss/rss-bot

```diff
@@ -82,6 +83,16 @@ parser.add_option('--feed-file',
                   help='The file containing a list of RSS feed URLs to follow, one URL per line',
                   default=os.path.join(RSS_DATA_DIR, "rss-feeds"),
                   action='store')
+parser.add_option('--unwrap',
+                  dest='unwrap',
+                  action='store_true',
+                  help='Convert word-wrapped paragraphs into single lines',
+                  default=False)
+parser.add_option('--math',
+                  dest='math',
+                  action='store_true',
+                  help='Convert $ to $$ (for KaTeX processing)',
+                  default=False)
```

Comment:
these should probably eventually go into the config file, not the command-line options.  But definitely an improvement, so I'll merge.